pom.xml

```diff
@@ -125,7 +126,7 @@
     <repositories>
         <!-- Try Maven central first, not last, which happens when omitted here -->
         <repository>
-            <id>central</id>
```

Comment:
I don't see any reason renaming this. The central repository id is often referred in local settings.xml (ie when configuring timeouts), this would need duplicit config with same content. So ... why?

Reply:
Because it conflicts when setting up multiple projects at the same time in settings.xml